src/main/java/persistence/sql/ddl/definition/ColumnDefinition.java

```diff
@@ -0,0 +1,67 @@
+package persistence.sql.ddl.definition;
+
+import jakarta.persistence.Column;
+import persistence.sql.ddl.query.CreateQueryBuilder.SQLTypeTranslator;
+
+import java.lang.reflect.Field;
+
+public class ColumnDefinition {
+    private final String name;
+    private final String type;
+    private final boolean nullable;
+    private static final int DEFAULT_LENGTH = 255;
```

Comment:
```suggestion
    private static final int DEFAULT_LENGTH = 255;
    
    private final String name;  
    private final String type;
    private final boolean nullable;
```
자바 컨벤션에서는 상수가 더 상위에 존재하여야합니다 :) 
https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html